code/modules/mob/living/simple_animal/friendly/butterfly.dm

```diff
@@ -31,3 +31,148 @@
 
 /mob/living/simple_animal/butterfly/bee_friendly()
 	return TRUE //treaty signed at the Beeneeva convention
+
+/mob/living/simple_animal/fly
+	name = "swarm of flies"
+	desc = "[health = 1 ? "A single brave little fly, ready to take what the world throws at it." : "A swarm of flies, because the Janitor just isn't doing a good enough job."]"
```

Comment:
this will only apply on Init(). make this update on take_damage

Reply:
Actually this probably wont work at all since you're not allowed to do things dynamically in prototypes

Reply:
why don't you epic men tell him to put it in examine()

Reply:
health == 1,maybe?

Reply:
yes

Reply:
all of the above